tests/php/Controllers/Tabbar/Navigation/Docs/DocsController/DocsControllerTopActionTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use App\Http\Controllers\Tabbar\Navigation\Docs\DocsController;
+use Illuminate\Http\Request;
+
+/**
+ * DocsControllerクラスのtopActionのテストを行う.
+ *
+ * @SuppressWarnings(PHPMD.TooManyPublicMethods)
+ */
+class DocsControllerTopActionTest extends TestCase
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)